packages/task/src/browser/task-frontend-contribution.ts

```diff
@@ -66,6 +66,12 @@ export namespace TaskCommands {
         category: TASK_CATEGORY,
         label: 'Clear History'
     };
+
+    export const TASK_TERMINATE: Command = {
+        id: 'task.terminate',
```

Comment:
[minor]: We could use `:` instead of `.` to align the new command ID with the existing ones.